src/Hyperopt.jl

```diff
@@ -157,7 +157,7 @@ function optimize(ho::Hyperoptimizer)
         if e isa InterruptException
             @info "Aborting hyperoptimization"
         else
-            rethrow(e)
```

Comment:
this will mess with the stacktrace and point to this line instead of the original one, so I fixed it to just `rethrow()` which does the right thing in a `catch` block

Reply:
Aha, I learned something new 😃